ndc-ir/src/main/kotlin/io/hasura/ndc/common/NDCScalars.kt

```diff
@@ -1,47 +1,136 @@
 package io.hasura.ndc.common
 
+import com.fasterxml.jackson.annotation.JsonValue
+
 enum class NDCScalar {
-    INT,
-    FLOAT,
     BOOLEAN,
     STRING,
-    DATETIME,
+    INT8,
+    INT16,
+    INT32,
+    INT64,
+    FLOAT32,
+    FLOAT64,
+    BIGINTEGER,
+    BIGDECIMAL,
+    UUID,
     DATE,
-    TIME,
-    TIME_WITH_TIMEZONE,
-    DATETIME_WITH_TIMEZONE
+    TIMESTAMP,
+    TIMESTAMPTZ,
+    GEOGRAPHY,
+    GEOMETRY,
+    BYTES,
+    JSON;
+
+    @JsonValue
```

Comment:
When I paired w/ @gneeri we noticed the result of `/schema` endpoint had UPPERCASE names, not sure if there is something going on here

Reply:
The lowercasing is only required on the type representation stuff. This should probably be refactored and cleaned up a bit.